README.md

```diff
@@ -64,12 +64,22 @@ The list of actions will likely grow as we get feedback from you.
 
 ## Install ⬇️
 
-### Running script (Recommended)
-
+### Running script (Recommended):
 ```bash
 bash <(curl -Ls https://install.tuist.io)
 ```
 
+
+### Using [Homebrew](http://brew.sh/):
+```
+brew install tuist
```

Comment:
Is there a formula for tuist on home-brew core?

Reply:
Removed this part